### DISCUSSION TITLE
Caption font size?

### DISCUSSION THREAD

**Discussion by kyledenlinger** [knowledgefutures/pubpub#2479] [Support]
Feels like recently the font size for image captions has gotten really large. Has something changed? 

Formerly:
![Screenshot at 2023-01-25 1539](https://user-images.githubusercontent.com/28068891/214685154-9cb52bd7-81e8-4140-b16e-abb215032bde.png)

Currently:
![Screenshot at 2023-01-25 1537](https://user-images.githubusercontent.com/28068891/214685206-5e6897fb-bf6c-4eb2-83bc-96043316507a.png)

I'm not an expert, but I think I'm comparing apples to apples here.

**Comment by gabestein** [comment]
Hrm, odd. Is one of those on a page vs. a pub? My guess is the two aren't equivalent (even though they probably should be).

**Comment by kyledenlinger** [comment]
I've seen the larger size on both pages and pubs. I started noticing it
earlier this week, thought it was due to some custom CSS I'd managed to
screw up, but it's also on projects without any custom CSS.

On Wed, Jan 25, 2023 at 4:06 PM Gabriel Stein ***@***.***>
wrote:

> Hrm, odd. Is one of those on a page vs. a pub? My guess is the two aren't
> equivalent (even though they probably should be).
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/pubpub/pubpub/discussions/2479#discussioncomment-4781833>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AGWEYG6TI2SRT2S4UU2QJWLWUGIWHANCNFSM6AAAAAAUGYSOLI>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>


-- 

Kyle Denlinger he/him/his

*Digital Pedagogy & Open Education Librarian*


Wake Forest University, Z. Smith Reynolds Library <https://zsr.wfu.edu/>

***@***.*** | meet with me > <https://calendly.com/kdenlinger>

**Reply by gabestein** [reply]
Hrm, thanks. I'm only seeing 15px so far on Pubs, but I am seeing 18 on pages. If you can point me to an example of a Pub with the larger caption size, that would be helpful.

**Reply by kyledenlinger** [reply]
Well, now that I'm looking into it, I can't find an instance of it being weird on a pub. Must be mistaken. Sorry about that. But I agree, captions should probably be equivalent on pubs and pages.
